node/src/GraphBuilder.cpp

```diff
@@ -151,15 +170,15 @@ namespace node {
     }
 
     Napi::Value GraphBuilder::Relu(const Napi::CallbackInfo& info) {
-        BUILD_UNARY(Relu);
+        BUILD_ACTIVATION_UNARY(Relu);
```

Comment:
With `BUILD_UNARY` and `BUILD_UNARY_OPERATOR`, it could be
```c++
if (info.Length() == 0) {
  BUILD_UNARY_OPERATOR(Relu);
} else {
  BUILD_UNARY(Relu);
}
```